constraints/utils.js

```diff
@@ -7,9 +7,10 @@ export const getStoredConstraints = (c) => c?.filter((c) => c.type == 'stored')
 export const hasStoredConstraints = (f) =>
   getStoredConstraints(f.constraints || []).length > 0
 export const getValidator = (v, n) => v.find((w) => w.validator === n)
-export const applyConstraintToRecord = (c, r, f) => {
+export const applyConstraintToRecord = (c, r, f, d) => {
+  const { config } = f.constraints?.find((fc) => fc.validator === c.validator)
   try {
-    eval(c.function)(r.get(f.key), f.key, { config: r._config, record: r })
+    eval(c.function)(r.get(f.key), f.key, { config, record: r, deps: d })
```

Comment:
Only thing to add here is a safeguard against a validator being in the blueprint that doesn't actually exist:
```
const constraints = f.constraints.filter((csnt) => csnt.validator === c.validator)
    if (constraints.length) {
      const config = constraints[0].config
      eval(c.function)(r.get(f.key), f.key, { config, record: r })
    } else {
      // If config couldn't be found, call w/o config
      eval(c.function)(r.get(f.key), f.key, { config: {}, record: r })
    }
```